spec/models/category_spec.rb

```diff
@@ -1,37 +1,37 @@
 require 'spec_helper'
 require 'shoulda-matchers'
 
+# this version of the method always makes vids one by one, as if each is a day older than the next created.  Refactor if more flexibility is required down the line.
 def assign_videos_to_category(int, category)
```

Comment:
`int` is not very descriptive. I would use something like `count` or `number_of_videos` instead.